sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ChatCompletionsJsonSchemaResponseFormatJsonSchema.java

```diff
@@ -34,8 +35,7 @@ public final class ChatCompletionsJsonSchemaResponseFormatJsonSchema
     /*
      * The schema property.
      */
-    @Generated
-    private Object schema;
+    private BinaryData schema;
```

Comment:
I think that we are shadowing what could potentially be an emitter issue by adding a code customization. I will reach out to see if this is actually behaving as expected.

Reply:
talked offline, will keep using BinaryData